src/casp/services/api/schemas/cellarium_general.py

```diff
@@ -1,11 +1,12 @@
 import datetime
+from typing import Optional
 
 from pydantic import BaseModel, Field
 
 
 class CASModel(BaseModel):
     model_name: str = Field(example="human-pca-001")
-    description: str = Field(example="1st version of PCA model for human data")
+    description: Optional[str] = Field(example="1st version of PCA model for human data")
```

Comment:
Conversely, we could also make sure to always return a `""` if it's empty - it simplifies handling on the client side I think.  Like, if there's an null, I think it would end up printing "None" in the description output I think